aikau/src/main/resources/alfresco/documentlibrary/_AlfCreateContentMenuItemMixin.js

```diff
@@ -0,0 +1,182 @@
+define(["dojo/_base/declare",
+        "alfresco/core/Core"], 
+        function(declare, AlfCore) {
+   
+   return declare([AlfCore], {
+
+      /**
+       * An array of the i18n files to use with this widget.
+       * 
+       * @instance
+       * @type {object[]}
+       * @default [{i18nFile: "./i18n/_AlfCreateContentMenuItemMixin.properties"}]
+       */
+      i18nRequirements: [{i18nFile: "./i18n/_AlfCreateContentMenuItemMixin.properties"}],
+
+      /**
+       * This is the type of content that will be created. This defaults to "cm:content" but can be overridden through
+       * configuration.
+       * 
+       * @instance
+       * @type {string}
+       * @default
+       */
+      contentType: "cm:content",
+
+      /**
+       * The title for the content creation dialog.
+       *
+       * @instance
+       * @type {String}
+       * @default
+       */
+      dialogTitle: "create-content-menu-item.dialog.title",
+
+      /**
+       * The label for the content creation dialog confirmation button.
+       *
+       * @instance
+       * @type {String}
+       * @default
+       */
+      dialogConfirmationLabel: "create-content-menu-item.dialog.confirmation",
+
+      /**
+       * The label for the content creation dialog cancellation button.
+       *
+       * @instance
+       * @type {String}
+       * @default
+       */
+      dialogCancellationLabel: "create-content-menu-item.dialog.cancellation",
+
+      /**
+       * The topic published when the content creation dialog is confirmed.
+       *
+       * @instance
+       * @type {String}
+       * @default
+       */
+      formSubmissionTopic: "ALF_CREATE_CONTENT_REQUEST",
+
+      /**
+       * This is the MIME type of the content that will be created. This defaults to "text/plain" but can be overridden
+       * through configuration.
+       *
+       * @instance
+       * @type {string}
+       * @default
+       */
+      mimeType: "text/plain",
+
+      /**
+       * This is the topic that will be published when the menu item is clicked. It defaults to the topic requesting
+       * a form dialog be displayed but can be overridden through configuration.
+       *
+       * @instance
+       * @type {String}
+       * @default
+       */
+      publishTopic: "ALF_CREATE_FORM_DIALOG_REQUEST",
+
+      /**
+       * It is expected that upload will require topics to be published globally, however this can be overridden through
+       * configuration.
+       *
+       * @instance
+       * @type {boolean}
+       */
+      publishGlobal: true,
+
+      /**
+       * This should be configured to be an array of form control widgets to be placed into the create content dialog.
+       *
+       * @instance
+       * @type {object[]}
+       * @default null
+       */
+      widgets: null,
+
+      /**
+       * 
+       * @instance
+       */
+      onClick: function alfresco_documentlibrary__AlfCreateContentPermissionsMixin__onClick(/*jshint unused:false*/ evt) {
```

Comment:
Incorrect function name for module `alfresco/documentlibrary/_AlfCreateContentMenuItemMixin`